// Copyright 2018 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file or at
// https://developers.google.com/open-source/licenses/bsd

// This file defines protobufs for issues and related business
// objects, e.g., field values, comments, and attachments.

syntax = "proto3";

package monorail;

// TODO(jojwang): monorail:1701, fill User with all info necessary for
// creating a user profile page.
// Next available tag: 5
message User {
  string email = 1;
  int64 user_id = 2;
  bool is_site_admin = 3;
  string availability = 4;
}


// Next available tag: 6
message Commit {
  string commit_sha = 1;
  int64 author_id = 2;
  fixed32 commit_time = 3;
  string commit_message = 4;
  string commit_repo_url = 5;
}


// Next available tag: 3
message UserPrefValue {
  string name = 1;
  string value = 2;
}
